holo-yang/modules/augmentations/holo-ospf.yang

```diff
@@ -240,21 +291,47 @@ module holo-ospf {
     }
   }
 
-   augment "/rt:routing/rt:control-plane-protocols/"
+  augment "/rt:routing/rt:control-plane-protocols/"
```

Comment:
nit: unnecessary white space change here and in a few other places

Reply:
those nits should be solved now